dev/README_RELEASE_PYTHON_CLIENT.md

```diff
@@ -140,8 +153,6 @@ git commit -m "Prepare release ${VERSION}${VERSION_SUFFIX}"
 cd ${AIRFLOW_REPO_ROOT}
 rm dist/*
 breeze release-management prepare-python-client --distribution-format both --python-client-repo "${CLIENT_REPO_ROOT}" --version-suffix ""
-breeze release-management prepare-tarball --tarball-type apache_airflow_python_client --version "${VERSION}" --version-suffix "${VERSION_SUFFIX}"
-
```

Comment:
The source package command is run after creating the tag below. Running it here doesn't work